app/javascript/components/public_health/PatientsTable.js

```diff
@@ -217,7 +217,7 @@ class PatientsTable extends React.Component {
         this.updateTable({ ...query, jurisdiction: jurisdictionId, page: 0 });
         this.updateAssignedUsers(jurisdictionId, this.state.query.scope, this.props.workflow, this.state.query.tab);
       }
-    } else if (event.target.name === 'assignedUser') {
+    } else if (event.target.name === 'assigned_user') {
```

Comment:
commenting on this line because github would only allow me to comment on changed lines, but would you mind also changing `jurisdictionId` to `jurisdiction_id` as well?

Reply:
done